src/component/Calculator.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable react/no-unused-state */
+/* eslint-disable no-mixed-spaces-and-tabs */
+/* eslint-disable no-tabs */
```

Comment:
* Kindly fix all eslint errors rather than inline disabling as the linters are there to help you write good and professional codes